### PR TITLE
Feature  agreement amendments redesign changes

### PR DESCRIPTION
**Pull request by adi130987**

### CODE REVIEW

**Comment by adi130987** [comment]
connects unicef/etools-issues#474

**Comment by adi130987** [comment]
connects unicef/etools-issues#490

File: pmp/app-elements/agreements/agreement-amendments.html [hunk 1]
```diff
@@ -215,26 +214,18 @@
               this.$$('#fileUpload_' + index).invalid = true;
```
Comment (acory) [inline pmp/app-elements/agreements/agreement-amendments.html]:
```
if (!amendm.signed_date) {
               this.$$('#signedDate_' + index).invalid = true;
               isValid = false;
             } else if (this._signedDateIsInTheFuture(amendm.signed_date)) {
               this.$$('#signedDate_' + index).errorMessage = 'Signed Date can not be in the future';
               this.$$('#signedDate_' + index).invalid = true;
               isValid = false;
}
```
can be replaced with :

```
let signedDateEl = this.$$('#signedDate_' + index);
signedDateEL.errorMessage = this._getSignedDateErrorMessage(amendm.signed_date);
isValid = signedDateEl.validate();
```

**Comment by adi130987** [comment]
connects unicef/etools-issues#479

**Pull request merged**